packages/protocol/test/governance/validators.ts

```diff
@@ -155,7 +155,7 @@ contract('Validators', (accounts: string[]) => {
 
   describe('#setMinElectableValidators', () => {
     const newMinElectableValidators = minElectableValidators.plus(1)
-    it('should set the minimum deposit', async () => {
+    it('should set the minimum commitment', async () => {
```

Comment:
Looks like this string was already wrong in previous commits, this should talk about minimum electable validators, right?